src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -635,7 +635,7 @@ public HttpResponse doUpdate(StaplerRequest req, StaplerResponse rsp) throws IOE
         j.clouds.replace(this, result);
         j.save();
         // take the user back to the cloud top page.
-        return FormApply.success("..");
+        return FormApply.success("../..");
```

Comment:
This should be followed up in Jenkins core, since in case you rename a cloud `..` will lead to an error, since the path has changed.